tools/build_chain.sh

```diff
@@ -1286,6 +1294,59 @@ check_bin()
     echo "Binary check passed."
 }
 
+prepare_ca(){
+    dir_must_not_exists ${output_dir}/chain
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5331982)